src/guide/contributing-to-an-open-jdk-project.md

```diff
@@ -32,6 +38,15 @@ Depending on your answer to that question you will need to consider one or more
 
 * **Modernizing** -- Changing code purely for the sake of using a new language feature isn't usually a good change. Be a subject matter expert, not just a language expert. Writing code in "a better way" is not guaranteed to be safe. A change of behavior is always possible and unless you understand the code you are changing at more than the core language/API level, and have looked into appropriate tests and can speak to the risks, then you should first find a subject matter expert to discuss it with. Keep in mind that the OpenJDK code is developed by a large community. If a new language feature is introduced, all developers working in that code must learn this new feature and understand the implications of using it.
 
+## I found an issue in JBS that I want to fix
+
+Great! This is a good way to get started with OpenJDK development. There are a few things you should think about before getting started.
+
+- As with any change, do bring it up for discussion on the [mailing lists](#mailing-lists) before spending time on the fix. Even though the issue has been filed in JBS there may be reasons not to fix that particular issue right now.
+- Never work on an issue that is assigned to someone else. Duplicating work is pointless. If you think that an issue should be fixed and it has been assinged to someone for a long time without progress, reach out to that person and ask what the current status is. Many issues requires a lot of thought, and there may be half-baked solutions that you can build on to avoid the traps the initial developer has already fallen into.
```

Comment:
```suggestion
- Never work on an issue that is assigned to someone else. Duplicating work is pointless. If you think that an issue should be fixed and it has been assigned to someone for a long time without progress, reach out to that person and ask what the current status is. Many issues requires a lot of thought, and there may be half-baked solutions that you can build on to avoid the traps the initial developer has already fallen into.
```